adminSiteClient/AdminApp.tsx

```diff
@@ -111,262 +121,284 @@ export class AdminApp extends React.Component<{
-                            <Route
-                                exact
-                                path="/charts/create"
-                                component={ChartEditorPage}
-                            />
-                            <Route
-                                exact
-                                path="/charts/:chartId/edit"
-                                render={({ match }) => (
-                                    <ChartEditorPage
-                                        grapherId={parseInt(
-                                            match.params.chartId
-                                        )}
-                                    />
-                                )}
-                            />
-                            <Route
-                                exact
-                                path="/charts/:chartId/edit/:config"
-                                render={({ match }) => (
-                                    <ChartEditorPage
-                                        grapherConfig={JSON.parse(
-                                            Base64.decode(match.params.config)
-                                        )}
-                                    />
-                                )}
-                            />
-                            <Route
-                                exact
-                                path="/charts"
-                                component={ChartIndexPage}
-                            />
-                            <Route
-                                exact
-                                path="/chartViews"
-                                component={ChartViewIndexPage}
-                            />
-                            <Route
-                                exact
-                                path="/chartViews/:chartViewId/edit"
-                                render={({ match }) => (
-                                    <ChartViewEditorPage
-                                        chartViewId={parseInt(
-                                            match.params.chartViewId
-                                        )}
-                                    />
-                                )}
-                            />
-                            <Route path="/images" component={ImageIndexPage} />
-                            <Route
-                                exact
-                                path={`/${EXPLORERS_ROUTE_FOLDER}/:slug`}
-                                render={({ match }) => (
-                                    <AdminLayout title="Create Explorer">
-                                        <ExplorerCreatePage
-                                            slug={match.params.slug}
-                                            gitCmsBranchName={gitCmsBranchName}
-                                            manager={admin}
+            <QueryClientProvider client={queryClient}>
```

Comment:
The changes below are mostly whitespace, except the new route and the new provider.